src/main/java/io/zonky/test/db/postgres/embedded/PreparedDbProvider.java

```diff
@@ -123,7 +123,7 @@ public ConnectionInfo createNewDatabase() throws SQLException
     public DataSource createDataSourceFromConnectionInfo(final ConnectionInfo connectionInfo) throws SQLException
     {
         final PGSimpleDataSource ds = new PGSimpleDataSource();
-        ds.setPortNumber(connectionInfo.getPort());
+        ds.setPortNumbers(new int[] {connectionInfo.getPort()});
```

Comment:
This change is not possible because the library should be compatible with a wider range of driver versions. If I use it with an older version of the driver, it causes the following error: `java.lang.NoSuchMethodError: org.postgresql.ds.PGSimpleDataSource.setServerNames([Ljava/lang/String;)V`

Reply:
reverted